### PR TITLE
Feature/sass

### PR DESCRIPTION
**Pull request by jcrain**
Use grunt-sass not grunt-contrib-sass. This saves the user from installing ruby to compile the sass.

### CODE REVIEW

**Comment by liorsaar1** [comment]
Update the doc.  ;)

File: package.json [hunk 1]
```diff
@@ -62,6 +61,7 @@
     "grunt-open": "~0.2.3",
     "grunt-postcss": "~0.7.1",
     "grunt-protractor-runner": "^2.0.0",
+    "grunt-sass": "^1.2.0",
     "grunt-usemin": "^3.0.0",
```
Comment (liorsaar1) [inline package.json]: test note

File: package.json [hunk 2]
```diff
@@ -62,6 +61,7 @@
...
     "grunt-wiredep": "^2.0.0",
```
Comment (liorsaar1) [inline package.json]: Another test note

**Comment by jcrain** [comment]
@liorsaar1 Should we merge?

**Pull request merged**